includes/Admin/Admin.php

```diff
@@ -229,4 +240,22 @@ public function prefill_post_title( $title ) {
 		// Otherwise it runs through wptexturize() and the like, which we want to avoid.
 		return $post->post_title;
 	}
+	
+	/**
+	 * Adds active publisher logo to media state output.
+	 *
+	 * @since 1.2.2
+	 *
+	 * @param string[] $media_states Array of media states.
+	 * @param WP_Post  $post    Post object.
+	 * @return string[] updated media states.
+	 */
+	public function media_states( $media_states, $post ): array {
```

Comment:
Could we add a simple unit test for this?

Reply:
Added a couple of tests

https://github.com/GoogleForCreators/web-stories-wp/pull/11836/commits/acdc0c2ccfeb2b127de73efd808f98d0913d5d0c